examples/llm/src/service.rs

```diff
@@ -73,23 +76,24 @@ impl Service for LlmService {
     type Parameters = ();
 
     async fn new(runtime: ServiceRuntime<Self>) -> Self {
-        LlmService { runtime }
+        let raw_weights = runtime
+            .fetch_url("https://huggingface.co/karpathy/tinyllamas/resolve/main/stories42M.bin");
```

Comment:
Awesome!
* Can we add some logging before the download starts?
* You probably need to update the README file (aka `lib.rs`)

Reply:
I updated the README, and I'll update logging in a separate PR :+1:

Reply:
Perhaps also create and link a task to make services even more long-lived?

Reply:
Done.